setup.py

```diff
@@ -41,5 +43,27 @@ def ensure_version():
         os.remove(version_filename)
 
 
+def generate_django_migrations():
+    from procrastinate.contrib.django.utils import generate_migrations
```

Comment:
Hm, this will import `procrastinate.__init__`, `procrastinate.contrib.__init__` and `procrastinate.contrib.django.__init__`. From an empty venv, this will very likely fail with `ImportError`. I think the generation code needs to live in the `setup.py` directly (or we need a top-level script that will be distributed alongside Procrastinate)